src/components/visualizations/chart/highcharts/test/getOptionalStackingConfiguration.spec.ts

```diff
@@ -118,20 +153,21 @@ describe('getOptionalStackingConfiguration', () => {
                 stackMeasuresToPercent: false,
```

Comment:
apart of stackMeasuresToPercent this expect is same as in next test case, why not to share series and yAxis?

Reply:
I fixed by using `it.each()` for this case